controllers/keystoneapi_controller.go

```diff
@@ -96,6 +98,7 @@ type KeystoneAPIReconciler struct {
 // +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete;
 // +kubebuilder:rbac:groups=route.openshift.io,resources=routes,verbs=get;list;watch;create;update;patch;delete;
 // +kubebuilder:rbac:groups=mariadb.openstack.org,resources=mariadbdatabases,verbs=get;list;watch;create;update;patch;delete;
+// +kubebuilder:rbac:groups=memcached.openstack.org,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Does the operator need `create;update;patch;delete` here?  Same with `list`.  Seems like it only `get`s and `watch`es the `Memcached` resource.

Reply:
This operator does not create or update Memcached resource. It requires that the user creates it seprately (eg via top-level control plane resource) and give the reference by memcachedInstance.

Reply:
Right.  That's why I'm asking if we can remove the `create;update;patch;delete` and `list` from the annotated permissions.  It seems we don't need them here.

Reply:
Sorry I thought I updated this but it was wrong. Fixed it in the latest version.

Reply:
fyi: I'm fixing this in horizon-operator by https://github.com/openstack-k8s-operators/horizon-operator/pull/168